CAM Files/4 Layer.cam

```diff
@@ -263,38 +263,39 @@
                         "index": 1,
                         "layer_type": "bottom"
                     },
-                    "filename_format": "%PREFIX/silkscreen_bottom.gbr",
+                    "filename_format": "%PREFIX/%N.GBO",
                     "layers": [
-                        22,
-                        26
+                        22
                     ],
                     "milling": false,
-                    "name": "Silkscreen Bottom",
+                    "name": "Bottom Silkscreen",
```

Comment:
For the Silkscreen layers, I see you only include tPlace/bPlace. Not tNames/bNames. Definitely the industry standard is to include reference designators from the names layers. And it looks like at least most HyTech boards also do this. So did you mean to exclude the names layers from silkscreen?